selfdrive/car/ford/fordcan.py

```diff
@@ -16,52 +16,54 @@ def create_lka_command(packer, angle_deg: float, curvature: float):
   values = {
     "LkaDrvOvrrd_D_Rq": 0,              # driver override level? [0|3]
     "LkaActvStats_D2_Req": 0,           # action [0|7]
-    "LaRefAng_No_Req": angle_deg,       # angle [-102.4|102.3] degrees
+    "LaRefAng_No_Req": 0,               # angle [-102.4|102.3] degrees
     "LaRampType_B_Req": 0,              # Ramp speed: 0=Smooth, 1=Quick
-    "LaCurvature_No_Calc": curvature,   # curvature [-0.01024|0.01023] 1/meter
+    "LaCurvature_No_Calc": 0,           # curvature [-0.01024|0.01023] 1/meter
     "LdwActvStats_D_Req": 0,            # LDW status [0|7]
     "LdwActvIntns_D_Req": 0,            # LDW intensity [0|3], shake alert strength
   }
   return packer.make_can_msg("Lane_Assist_Data1", CANBUS.main, values)
```

Comment:
@incognitojam can we just create a message with an empty dict, or will we ever use this message in the future?